remover:  
    # parameters of ROI(Region of Interest)
    L_max: 60.0
    min_h: -0.5
    max_h: 2.5
    # parameters of Group-based and Cell-based Zoning Model
    N_y: 30  
    N_x: 60
    N_c: 240
    # parameters of Group-based Region Detection(OHR,FOD,AOR) 
    minimum_num_pts: 10
    Thres: 5
    Kappa: 2   
    # parameters of ground fitting
    gf_dist_thr: 0.05
    gf_iter_times: 3
    gf_mini_num_points: 10
    gf_seeds_heigt_thr: 0.2

MapUpdater:
    dataset_name: "POSS"
    sequence_name: "00"
    initial_map_path: "/home/zgy/ROS/REMOVER/src/DR-REMOVER-master/DR-REMOVER-master/MAP/00_0_to_370_w_interval1_voxel_0.200000.pcd"
    save_path: "/home/zgy/ROS/REMOVER/src/DR-REMOVER-master/DR-REMOVER-master/results/"
    query_voxel_size: 0.2
    map_voxel_size: 0.2  
    removal_interval: 1

tf:
     tf_lidar2body: [1.0, 0.0, 0.0, 0.0,
                     0.0, 1.0, 0.0, 0.0,
                     0.0, 0.0, 1.0, 2.15,
                     0.0, 0.0, 0.0, 1.0]

